crates/bevy_transform/Cargo.toml

```diff
@@ -8,10 +8,15 @@ repository = "https://github.com/bevyengine/bevy"
 license = "MIT OR Apache-2.0"
 keywords = ["bevy"]
 
+[features]
+serialize = ["serde"]
+
 [dependencies]
 # bevy
 bevy_app = { path = "../bevy_app", version = "0.9.0-dev" }
 bevy_ecs = { path = "../bevy_ecs", version = "0.9.0-dev", features = ["bevy_reflect"] }
 bevy_hierarchy = { path = "../bevy_hierarchy", version = "0.9.0-dev" }
 bevy_math = { path = "../bevy_math", version = "0.9.0-dev" }
```

Comment:
Carrying this through, the serialize implementations of these types are reliant on serialization of bevy_math's and, by proxy, glam's serialization features. Probably need to pass this through as an optional feature there.

Reply:
`bevy_math`'s types already implement `Serialize`. It's not behind a feature. Should I make it optional there?

Reply:
Yes, please. The impact of removing serde from bevy_transform isn't beneficial if bevy_math and glam depend on it.

Reply:
done.

Reply:
Making `Serialize` optional in `bevy_math` seems to cause issues in `bevy_reflect`, which expects `Rect` to be `Serialize`.

Reply:
> Perhaps it is expected that bevy_math's types are always Serialize though?

We don't want this: there are users who want to use their own serialization solution (or don't care about it at all), and would really like to avoid pulling in serde anywhere :)